src/components/settings/sections/models/ChatModelsSubSection.tsx

```diff
@@ -52,6 +72,19 @@ export function ChatModelsSubSection({
                 <td>{chatModel.id}</td>
                 <td>{chatModel.providerId}</td>
                 <td>{chatModel.model}</td>
+                <td>
+                  <div
+                    className={clsx('checkbox-container', {
+                      'is-enabled': isEnabled(chatModel.enable),
+                    })}
+                    onClick={() => handleToggleEnableChatModel(chatModel.id)}
+                  >
+                    <input
+                      type="checkbox"
+                      checked={isEnabled(chatModel.enable)}
+                    />
+                  </div>
```

Comment:
You can use https://github.com/glowingjade/obsidian-smart-composer/blob/main/src/components/common/ObsidianToggle.tsx component for this

Reply:
> You can use https://github.com/glowingjade/obsidian-smart-composer/blob/main/src/components/common/ObsidianToggle.tsx component for this

Thank you for the suggestion! I've updated the code to use the ObsidianToggle component and simplified the logic by removing the negation.